cinder/mkdocs_theme.yml

```diff
@@ -2,3 +2,12 @@ cinder_theme: true
 
 static_templates:
   - 404.html
+
+include_search_page: false
+search_index_only: false
```

Comment:
and this one?

I realize these are in the upstream theme files but I don't understand what we are changing here.

Reply:
Copied from: https://github.com/mkdocs/mkdocs/blob/master/mkdocs/themes/mkdocs/mkdocs_theme.yml#L7

You can read what it does here: https://www.mkdocs.org/user-guide/custom-themes/#search_index_only

Reply:
> When search_index_only is set to false, then the search plugin modifies the Jinja environment by adding its own templates directory (with a lower precedence than the theme) and adds its scripts to the extra_javascript config setting.
>
>When search_index_only is set to true or not defined, the search plugin makes no modifications to the Jinja environment. A complete solution using the provided index file is the responsibility of the theme.

Reply:
It sounds like this maintains the previous default behavior.  We are not overriding default search index behavior in the theme.